src/rgw/rgw_lc_s3.cc

```diff
@@ -139,7 +139,16 @@ void LCFilter_S3::decode_xml(XMLObj *obj)
     single_cond = true;
   }
 
-  RGWXMLDecoder::decode_xml("Prefix", prefix, o);
+  {
+    auto prefix_iter = o->find("Prefix");
+    auto prefix_val = prefix_iter.get_next();
+    if(prefix_val) {
```

Comment:
probably invalid if prefix_iter is invalid, should be `prefix_iter && prefix_val`